app/javascript/listings/components/CategoryLinks.jsx

```diff
@@ -8,6 +8,7 @@ const CategoryLinks = ({ categories, onClick, selectedCategory }) => {
         <a
           href={`/listings/${category.slug}`}
           id={`category-link-${category.slug}`}
+          data-testid={category.slug === selectedCategory ? 'selected-category' : ''}
```

Comment:
Is the goal here to just have a `data-testid` when `category.slug` exists? If so, you can do this instead so you don't end up with a `data-testid` as an empty string.

```javascript
      {categories.map((category) => {
        const dataTestIdProp =
          category.slug === selectedCategory
            ? { 'data-testid': 'selected-category' }
            : {};

        return (
          <a
            href={`/listings/${category.slug}`}
            id={`category-link-${category.slug}`}
            className={category.slug === selectedCategory ? 'selected' : ''}
            onClick={(e) => onClick(e, category.slug)}
            data-no-instant
            {...dataTestIdProp}
          >
            {category.name}
          </a>
        );
      })}
```

I see this down below as well in other tests, so same thing.

Reply:
✅